scheds/rust/scx_layered/src/bpf/util.bpf.c

```diff
@@ -1,19 +1,23 @@
-/* to be included in the main bpf.c file */
-
 #ifdef LSP
+#ifndef __bpf__
 #define __bpf__
-#ifndef LSP_INC
+#endif
+#define LSP_INC
 #include "../../../../include/scx/common.bpf.h"
+#else
+#include <scx/common.bpf.h>
 #endif
 
-#include "intf.h"
-
+#include <errno.h>
 #include <stdbool.h>
 #include <string.h>
 #include <bpf/bpf_core_read.h>
 #include <bpf/bpf_helpers.h>
 #include <bpf/bpf_tracing.h>
-#endif
+
+#include "intf.h"
+#include "timer.bpf.h"
```

Comment:
Why does util need to include `timer.bpf.h`, there's no timer being used at all IIRC.